qiskit/algorithms/fidelities/fidelity_result.py

```diff
@@ -0,0 +1,32 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Fidelity result class
+"""
+
+from __future__ import annotations
+
+from typing import Any
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class FidelityResult:
+    """Result of Fidelity computation.
+
+    Args:
+        values (list[float]): List of fidelity values for each pair of input circuits.
+        metadata: Additional information on the fidelity calculations.
+    """
+
+    values: list[float]
+    metadata: list[dict[str, Any]]
```

Comment:
Is there a difference between using the python built-ins (list) or the `typing` objects (List)? 🤔

Reply:
I think its preferred nowadays to use the builtins since they are supported as typehints now and no longer use List, Dict ec

Reply:
Semantically no difference As Steve said, Python version issue.